conf/application.conf

```diff
@@ -35,6 +35,13 @@ akka {
   #log-config-on-start = true
 }
 
+## Enable HTTPS
+# https://www.playframework.com/documentation/latest/ConfiguringHttps
+# ~~~~~
+# By default, Play listens for unencrypted HTTP traffic on port 9000.
+# gRPC requires HTTP/2, which in turn requires an encrypted connection.
+play.server.https.port = 9443
```

Comment:
oh so the `-Dhttps.port` I found somewhere was misleading i guess... okey thanks!